docs/devguide/event-conventions.asciidoc

```diff
@@ -36,6 +37,26 @@ workers.idle
 - Do not use dots (.) in individual field names. Dots are reserved for grouping related fields into subdocuments. 
 - Use singular and plural names properly to reflect the field content. For example, use `requests_per_sec` rather than `request_per_sec`. 
 
+[[units]]
+==== Units
+
+These are well-known suffixes to represent units of stored values, use them as a dotted suffix when
+possible. For example `system.memory.total.bytes` or `system.diskio.read.count`:
```

Comment:
Should we really use an example here we don't follow yet? :-)

Reply:
not sure I understand, I think I took this metrics from the actual module. Could you please explain a little more?

Reply:
Here it seems it's `system.memory.total` and not `system.memory.total.bytes` or do I miss something? https://github.com/elastic/beats/blob/master/metricbeat/module/system/memory/_meta/fields.yml#L7 See also https://github.com/elastic/beats/blob/master/metricbeat/module/system/memory/_meta/data.json#L47

Reply:
gotcha, definitely bad example :see_no_evil:, updated it